pkg/operation/botanist/component/metricsserver/metrics_server_test.go

```diff
@@ -271,6 +272,35 @@ spec:
         volumeMounts:
         - mountPath: /srv/metrics-server/tls
           name: metrics-server
+      - name: metrics-server-nanny
+        image: ` + sideCar + `
+        resources:
+          limits:
+            cpu: 40m
+            memory: 25Mi
+          requests:
+            cpu: 40m
+            memory: 25Mi
+        env:
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+        command:
+          - /pod_nanny
+          - --cpu=20m
+          - --extra-cpu=1m
+          - --memory=15Mi
+          - --extra-memory=2Mi
+          - --threshold=5
+          - --deployment=metrics-server
+          - --container=metrics-server
+          - --poll-period=300000
+          - --use-metrics=true
```

Comment:
```suggestion
          - --use-metrics=false
```
Shouldn't this be `false` if it is `false` in `metrics_server.go`?